trigger/bme280streamrpi/bme280lib.go

```diff
@@ -0,0 +1,241 @@
+package bme280streamrpi
+
+// Original file coming from https://github.com/taiyoh/go-embd-bme280/blob/master/bme280.go
```

Comment:
is it possible to ref the package rather than copying it?

Reply:
No, because the DeviceAddr in the original library is not correct (0x76). 
Correct one is 0x77.